jsonnet/components/telemeter-client.libsonnet

```diff
@@ -19,15 +19,23 @@ function(params) {
       ],
     },
   },
+  local commonLabels = {
+    metadata+: {
+      labels+: cfg.commonLabels,
+    },
+  },
 
   // Remapping everything as this is the only way I could think of without refactoring imported library
-  prometheusRule: tc.telemeterClient.prometheusRule,
-  clusterRoleBindingView: tc.telemeterClient.clusterRoleBindingView,
-  clusterRoleBinding: tc.telemeterClient.clusterRoleBinding,
-  clusterRole: tc.telemeterClient.clusterRole,
-  serviceAccount: tc.telemeterClient.serviceAccount,
-  service: tc.telemeterClient.service,
+  prometheusRule: tc.telemeterClient.prometheusRule + commonLabels,
```

Comment:
I chose this over a function that will implicitly inject the labels.

(better solution is to make the upstream take care of it)

Reply:
we chose to inject at the end